hyundai_kia_connect_api/VehicleManager.py

```diff
@@ -88,6 +88,30 @@ def check_and_refresh_token(self) -> bool:
             return True
         return False
 
+    def start_climate(self, vehicle_id: str, options: ClimateRequestOptions) -> str:
+        return self.api.start_climate(self.token, self.get_vehicle(vehicle_id), options)
+            
+    def stop_climate(self, vehicle_id: str) -> str:
+        return self.api.stop_climate(self.token, self.get_vehicle(vehicle_id))
+
+    def lock(self, vehicle_id: str) -> str:
+        return self.api.lock_action(self.token, self.get_vehicle(vehicle_id), "close")
```

Comment:
Magical strings are evil :-) let's use ENUM from const.py

Reply:
Are you able to show me what that would look like?  Sounds like another concept for me to google.  :)

Reply:
As a starting point, we can return a id as you are doing now and create another function to check its status, not an urgent but it will make our life a lot of easier for HA, and anyone can use it inside async scope.

Reply:
Sorry, my comment was about rather than passing `close` or `open`, can we pass LockAction.Open and LockAction.Close which are in fact defined in const.py

Reply:
Should be resolved now.